extensions/adobe/experience/customerJourneyManagement/ajo-entity-class.schema.json

```diff
@@ -0,0 +1,32 @@
+{
+  "meta:license": [
+    "Copyright 2022 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/ajo/entity",
```

Comment:
Since this is a n Adobe Solution specific class, could you please change the $id like below:
"https://ns.adobe.com/experience/customerJourneyManagement/ajoEntity"